src/app/modules/main/chat_section/chat_content/view.nim

```diff
@@ -12,6 +12,9 @@ QtObject:
       chatDetails: ChatDetails
       chatDetailsVariant: QVariant
       
+  # Forward declaration
+  proc chatDetailsChanged*(self:View)
```

Comment:
@jrainville this forward declaration for a signal looks a bit weird (signal is already just a single line declaration/definition) please just move line 98 here.